examples/notion-task-github-pr-sync/index.js

```diff
@@ -0,0 +1,140 @@
+/* ================================================================================
+
+	notion-task-github-pr-sync
+  
+  Glitch example: https://glitch.com/edit/#!/notion-github-sync
+  Find the official Notion API client @ https://github.com/makenotion/notion-sdk-js/
+
+================================================================================ */
+
+function print(x) {console.log(x)}
+const { Client } = require("@notionhq/client")
+const dotenv = require("dotenv")
+const { Octokit } = require("octokit")
+const _ = require("lodash")
+
+dotenv.config()
+const octokit = new Octokit({ auth: process.env.GITHUB_KEY })
+const notion = new Client({ auth: process.env.NOTION_KEY })
+
+const databaseId = process.env.NOTION_DATABASE_ID
+const OPERATION_BATCH_SIZE = 10
+
+/**
+ * Entry Point
+ */
+updateNotionDBwithGithubPRs();
+
+/**
+ * Fetches PRs from Github and updates the according Notion Task
+ */
+async function updateNotionDBwithGithubPRs(){
+  const prs = await fetchPRsFromGitHub();
+  const prsToUpdate = []  
+  for (pr of prs){
+    if (! await hasIntegrationCommentedOnPage(pr.page_id)) {
+      prsToUpdate.push(pr);
+    }
+  }
+  updatePages(prsToUpdate);
+}
+
+/**
+ * Returns whether integration has commented 
+ * @params page_id: string 
+ * @returns {Promise<Boolean>}
+ */
+async function hasIntegrationCommentedOnPage(page_id){ 
+  const commentedUsers = [];
+  const comments = await notion.comments.list({ block_id: page_id });
+  const bot = await notion.users.me()
+  if (comments.results) {
+    for (const comment of comments.results){ 
+      commentedUsers.push(comment.created_by.id);
+    }
+    if (commentedUsers.includes(bot.id) && commentedUsers != []) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+}
+
+/**
+ * Fetch PRs from Github 
+ * @returns {Promise<Array<{ title: string, task_link: string, state: "open" | "closed"}>>}
+ */
+async function fetchPRsFromGitHub() {
```

Comment:
nit: this function is a little redundant because it only calls `getGitHubPRsForRepository`, can we remove it?